### DISCUSSION TITLE
Is there any way to turn PiVPN off?

### DISCUSSION THREAD

**Discussion by melerodev** [pivpn/pivpn#1671] [Support]
Hi, I need to know if there is a way to turn off the VPN with commands. I have read the [documentation ](https://docs.pivpn.io/wireguard/)of the commands but this one doesn't say anything about this, it only says how to delete and some other things.

**Comment by coolapso** [comment]
Before opening any issue on any opensource project, its always good idea to read both the documentation and the README file, you should have read this: https://github.com/pivpn/pivpn#feedback--support and followed the instructions. 

Moving to discussions. 

To answer your question. 

To stop and disable: 

OpenVPN: 

```
sudo systemctl stop openvpn.service
sudo systemctl disable openvpn.service
```

wireguard: 

```
sudo systemctl stop wireguard@wg-0
sudo systemctl disable wireguard@wg-0
```

**Reply by melerodev** [reply]
> Before opening any issue on any opensource project, its always good idea to read both the documentation and the README file, you should have read this: https://github.com/pivpn/pivpn#feedback--support and followed the instructions.
> 
> Moving to discussions.
> 
> To answer your question.
> 
> To stop and disable:
> 
> OpenVPN:
> 
> ```
> sudo systemctl stop openvpn.service
> sudo systemctl disable openvpn.service
> ```
> 
> wireguard:
> 
> ```
> sudo systemctl stop wireguard@wg-0
> sudo systemctl disable wireguard@wg-0
> ```

Thank you very much for answering and for your help and forgive my ignorance.
